src/geshi/lisp.php

```diff
@@ -52,30 +52,301 @@
+            'storage-condition','store-value', 'stream', 
+            'stream-element-type','stream-error', 
+            'stream-error-stream','stream-external-format', 'streamp', 
+            'streamup','string', 'string-capitalize', 'string-char', 
+            'string-char-p','string-downcase', 'string-equal', 
+            'string-greaterp','string-left-trim', 'string-lessp', 
+            'string-not-equal','string-not-greaterp', 
+            'string-not-lessp','string-right-strim', 
+            'string-right-trim','string-stream', 'string-trim', 
+            'string-upcase','string/=', 'string<', 'string<=', 
+            'string=','string>', 'string>=', 'stringp', 'structure', 
+            'structure-class','structure-object', 'style-warning', 
+            'sublim','sublis', 'subseq', 'subsetp', 'subst', 
+            'subst-if','subst-if-not', 'substitute', 'substitute-if', 
+            'substitute-if-not','subtypep', 'svref', 'sxhash', 
+            'symbol','symbol-function', 'symbol-macrolet', 
+            'symbol-name','symbol-package', 'symbol-plist', 
+            'symbol-value','symbolp', 'synonym-stream', 
+            'synonym-stream-symbol','sys', 'system', 't', 'tagbody', 
+            'tailp','tan', 'tanh', 'tenth', 'terpri', 'the', 'third', 
+            'throw','time', 'trace', 'translate-logical-pathname', 
+            'translate-pathname','tree-equal', 'truename', 'truncase', 
+            'truncate','two-way-stream', 
+            'two-way-stream-input-stream',
+            'two-way-stream-output-stream','type', 'type-error', 
+            'type-error-datum','type-error-expected-type', 'type-of', 
+            'typecase','typep', 'unbound-slot', 
+            'unbound-slot-instance','unbound-variable', 
+            'undefined-function','unexport', 'unintern', 'union', 
+            'unless','unread', 'unread-char', 'unsigned-byte', 
+            'untrace','unuse-package', 'unwind-protect', 
+            'update-instance-for-different-class',
+            'update-instance-for-redefined-class',
+            'upgraded-array-element-type',
+            'upgraded-complex-part-type','upper-case-p', 
+            'use-package','use-value', 'user', 
+            'user-homedir-pathname','value', 'values', 'values-list', 
+            'variable','vector', 'vector-pop', 'vector-push', 
+            'vector-push-extend','vectorp', 'warn', 'warning', 'when', 
+            'wild-pathname-p','with-accessors', 
+            'with-compilation-unit','with-condition-restarts', 
+            'with-hash-table-iterator','with-input-from-string', 
+            'with-open-file','with-open-stream', 
+            'with-output-to-string','with-package-iterator', 
+            'with-simple-restart','with-slots', 
+            'with-standard-io-syntax','write', 'write-byte', 
+            'write-char','write-line', 'write-sequence', 
+            'write-string','write-to-string', 'y-or-n-p', 
+            'yes-or-no-p','zerop'
```

Comment:
Please note source formatting of existing files.
Comma is followed by single whitespace except @ EOL.